README.md

```diff
@@ -10,7 +10,7 @@ RethinkDNS runs `serverless-dns` in production at these endpoints:
 |--------------------|------------------|-------------|---------------------------|-----------------------------------------|
 | ⛅ Cloudflare Workers | 200+ ([ping](https://check-host.net/check-ping?host=https://sky.rethinkdns.com))           | DoH         | `sky.rethinkdns.com`    | [configure](https://rethinkdns.com/configure?p=doh)  |
 | 🦕 Deno Deploy        | 30+ ([ping](https://check-host.net/check-ping?host=https://deno.dev))                      | DoH         | _private beta_          |                                         |
-| Fastly Compute@Edge   | 80+ ([ping](https://check-host.net/check-ping?host=https://serverless-dns.edgecompute.app))| DoH         | _private beta_          | [configure](https://rethinkdns.com/configure?p=doh) |
+| 🕐 Fastly Compute@Edge   | 80+ ([ping](https://check-host.net/check-ping?host=https://serverless-dns.edgecompute.app))| DoH         | _private beta_          | [configure](https://rethinkdns.com/configure?p=doh) |
```

Comment:
We sure... A clock? ;)

Reply:
They have a clock in the logo so I think it would be fit. But this one fits better with the logo⏱️

Reply:
After speaking with a colleague at Fastly, we thought a clock was best ☺️  I agree with @GiddyGoatGaming that it should be ⏱️

Reply:
👍